dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/MetadataSchemaRestRepository.java

```diff
@@ -53,10 +52,10 @@ public MetadataSchemaRest findOne(Context context, Integer id) {
     }
 
     @Override
-    public Page<MetadataSchemaRest> findAll(Context context, Pageable pageable) {
+    public Page<MetadataSchemaRest> findAll(Pageable pageable) {
```

Comment:
Question: for documentation purposes, should all endpoint methods have an authorization annotation (including ones allowed by anonymous) to make the permissions very explicit?

Reply:
Good point, however I think that people we easily forget the `@PreAuthorize("hasAuthority('ANONYMOUS')")` as it will have no effect.